CITATION.cff

```diff
@@ -0,0 +1,22 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Resende"
+  given-names: "Ximenes"
+- family-names: "de Sá"
+  given-names: "Fernando Henrique"
```

Comment:
alphabetical order.

Reply:
I have very little contribution to this package, in any sense. maybe we should rearrange the order of the authors.